### DISCUSSION TITLE
There should be a command changed.

### DISCUSSION THREAD

**Discussion by pingguomc** [Zrips/CMI#8043] [Enhancement]
command spy may lead to username leakage

How to leak:

1. The server uses login and cmi plugins

2. An OP used the command spy

3. A player joined the server

4. This OP will see: [Command Spy] player:/login His password

If maliciously exploited, it may result in user password leakage, leading to Credential Stuffing Attack

My suggestion : CommandSpy automatically blocks login plugin instructions

**Comment by Zrips** [comment]
There is option in config file to define blacklisted commands which will be ignored unless you have bypass permission node. I could remove that option to bypass with permission node to completely hide blacklisted commands
